fold-and-r.csl

```diff
@@ -129,6 +129,15 @@
             </group>
           </group>
         </else-if>
+        <else-if type="speech">
+          <group delimiter=" ">
+            <text macro="title" quotes="true"/>
+            <group suffix=".">
+              <text variable="event" prefix=", at "/>
```

Comment:
same here, actually -- you can't combine a space in a delimiter with a comma in a prefix. Here I'd remove the delimiter, which doesn't acutally do anything anymore.